fortuna/package-lock.json

```diff
@@ -5,20 +5,28 @@
   "requires": true,
   "dependencies": {
     "@babel/cli": {
-      "version": "7.7.0",
```

Comment:
This stuff should be added to our .gitignore so it doesn't show up here

Reply:
I'll just revert the commit making the update to the package-lock. **However** I do have an issue with us ignoring it all-together. On that note, I'll bring that discussion to the Discord.